### ISSUE TITLE
AssertionError: Trying to copy from a source that does not exist ( .gitignore )

### ISSUE THREAD

**Issue by vipulkrsingh** [hasura/generator-hasura-web#7]
I ran

`npm install -g yo`
`npm install -g generator-hasura-web`
`yo hasura-web`

Throws up following error

```
Creating Example app on the current directory.

events.js:160
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: /usr/local/lib/node_modules/generator-hasura-web/generators/app/templates/app/.gitignore
    at EditionInterface.exports._copySingle (/usr/local/lib/node_modules/generator-hasura-web/node_modules/mem-fs-editor/lib/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/usr/local/lib/node_modules/generator-hasura-web/node_modules/mem-fs-editor/lib/actions/copy.js:23:17)
    at child.tplCopy (/usr/local/lib/node_modules/generator-hasura-web/generators/app/index.js:7:11)
    at files.forEach.filename (/usr/local/lib/node_modules/generator-hasura-web/generators/app/index.js:17:5)
    at Array.forEach (native)
    at child.fileListCopy (/usr/local/lib/node_modules/generator-hasura-web/generators/app/index.js:16:9)
    at child.writing (/usr/local/lib/node_modules/generator-hasura-web/generators/app/index.js:44:5)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-hasura-web/node_modules/yeoman-generator/lib/base.js:431:23)
    at /usr/local/lib/node_modules/generator-hasura-web/node_modules/run-async/index.js:25:25
    at /usr/local/lib/node_modules/generator-hasura-web/node_modules/run-async/index.js:24:19
```

PS -> creating .gitignore file in the reported folder fixes the issue
PS -> Following the steps mentioned in On Development section of readme generates the app fine.

**Comment by coco98** [comment]
@vipulkrsingh I think an older version has gotten pushed.
Thanks for your pull request by the way 👍

**Comment by coco98** [comment]
Fixed in 0.3.0-rc2

**Comment by coco98** [comment]
@vipulkrsingh Could you try testing again? Everything should work now.
